doc/book/examples.md

```diff
@@ -171,6 +171,7 @@ class CacheListener implements ListenerAggregateInterface
     use ListenerAggregateTrait;
 
     private $cache;
+    private $listeners = [];
```

Comment:
You can force push to the same branch after a revert or rebase.

On Aug 18, 2016 4:33 PM, "Adam" <notifications@github.com> wrote:

> In doc/book/examples.md
> <https://github.com/zendframework/zend-eventmanager/pull/32#discussion_r75393488>
> :
>
> > @@ -171,6 +171,7 @@ class CacheListener implements ListenerAggregateInterface
> >      use ListenerAggregateTrait;
> >
> >      private $cache;
> > +    private $listeners = [];
>
> @webimpress <https://github.com/webimpress> Done, with another commit, do
> you know if I can revert commits on Github web without cloning and
> accessing CLI?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendframework/zend-eventmanager/pull/32/files/89559865b7d7169bf3b06c3e8b59d9526f59bbb8#r75393488>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABlV0AtfrImPm2q9KLMTmO4V2bcYWWfks5qhM-7gaJpZM4Jm8kp>
> .
>